src/evrythng/tests/test_entities.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+
+from evrythng.entities import actions
+from evrythng.entities import products
+
+
+class TestProductMethods(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.product_id = products.create_product(
```

Comment:
Will you please add """Product Tests Setup""" here and then the other missing doc strings?  The """ is more consistent with the rest of our code base too.

Check [this](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) out for information on doc string conventions.  Though ''' are in fact needed when you have a """ within the doc string.